tests/integration/wallet/test_commands.py

```diff
@@ -79,9 +83,13 @@ def shutdown(self):
     def send_claim_action(self, action):
         pass
 
+    def send_credits_sent(self):
+        pass
+
 
 class CommandTestCase(IntegrationTestCase):
 
+    timeout = 300
```

Comment:
Why does this need to change?

Reply:
Earlier I thought the test was failing due to timeout(the tests take 90 or so secs now btw) so I increased it and kept it that way. I can remove this if needed. Also this helps in debugging. Though I have no strong opinions.

Reply:
How long does it currently take to run the test? If we are close to 120 already then updating is fine, if we're still far from 120 then let's just wait (and maybe even make the test run faster).